Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Internal/BulkLocUpdate.hs

```diff
@@ -55,5 +59,12 @@ bulkLocUpdate req = do
       then Just <$> TM.getServiceConfigForRectifyingSnapToRoadDistantPointsFailure booking.providerId booking.merchantOperatingCityId
       else pure Nothing
   let isTollApplicable = DC.isTollApplicableForTrip booking.vehicleServiceTier booking.tripCategory
-  _ <- addIntermediateRoutePoints defaultRideInterpolationHandler rectificationServiceConfig isTollApplicable transportConfig.enableTollCrossedNotifications rideId driverId loc
+  let passedThroughDrop = any (isDropInsideThreshold booking transportConfig) loc
```

Comment:
Can we move this logic inside LocationUpdate library as then it would be handled in both bulk location update and rideEnd case

Reply:
but this logic is not required in endRide ?